### PR TITLE
Master skills dashboard

### LINKED ISSUE #8: An admin can add new master skill details
For now, let's assume this means they're doing so via ActiveAdmin. This ticket should be the result of us creating the `OccupationStandard` table and then linking it up to ActiveAdmin.

"Masker skill details" really means an occupation_standard that's a "framework"

It should also include the actual skills / work processes associated with that `OccupationStandard`: https://dbdiagram.io/d/5d7aa7d683427516dc0b9bd6

### LINKED ISSUE #9: An admin can delete master skill details
Also likely covbered by https://github.com/WorkHands/RAPiDSkills/issues/8 with active admin.

It should also include the actual skills / work processes associated with that `OccupationStandard`: https://dbdiagram.io/d/5d7aa7d683427516dc0b9bd6

### LINKED ISSUE #10: An admin can update master skill details
This'll likely be covered by https://github.com/WorkHands/RAPiDSkills/issues/8 by default with active admin

It should also include the actual skills / work processes associated with that `OccupationStandard`: https://dbdiagram.io/d/5d7aa7d683427516dc0b9bd6

### PR DESCRIPTION
**Pull request by littleforest**
closes #8 
closes #9 
closes #10

* Adds Industry model
* Adds Organization model
* Adds OrganizationStandard model with STI for Framework, Registered, Guideline, Unregistered submodels

Also adds following models with associations:
* locations
* states
* standards_registration
* work_processes
* skills
* occupation_standard_skills

All added to ActiveAdmin

### CODE REVIEW

File: app/models/registered.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Registered < OccupationStandard
```
Comment (Pcushing) [inline app/models/registered.rb]: I'm thinking these `type` names might get confusing without more context. 

How about `RegisteredStandard`, `UnregisteredStandard`, etc.?

**Comment by Pcushing** [comment]
Re:

`
closes #8 
closes #9 
closes #10
`

I think the issue descriptions arent great / don't cover the entirety of what was implied. Those tickets should encompass the adding / removinng of skills annd work processes. I'm goingn to update them. Also, I think we should make a join between `OccupationStandard` and `WorkProcess` so we can see how these are re-used across standards, similar to with skills.

I'm going to update those tickets. As a next steps, let's knock out those two join tables and the work processes / skills tables.

I updated the schema, too: https://dbdiagram.io/d/5d7aa7d683427516dc0b9bd6

File: app/models/registered.rb (hunk 1 above)
Comment (littleforest) [reply app/models/registered.rb]: Yep, agreed!  I'll make that change.

**Comment by littleforest** [comment]
@Pcushing would it be easiest if I just merged the code for #50 into this PR?

**Comment by Pcushing** [comment]
> @Pcushing would it be easiest if I just merged the code for #50 into this PR?

Works for me if that's easier for you!

**Comment by littleforest** [comment]
@Pcushing ready for re-review.

**Comment by Pcushing** [comment]
👍 good to go

**Pull request merged**